helm/ph-ee-engine/ams-paygops/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ph-ee-connector-ams-paygops
+  namespace: paymenthub
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ph-ee-connector-ams-paygops
```

Comment:
why the role name is same as the service name?

Reply:
This is serviceaccount name

Reply:
add `-role`  at the end as per the reference

Reply:
sample {{ template "<>.serviceAccount" . }}-role

Reply:
adding role only when I use template names. since I hardcoded role name need not to add -role